nix-build-cache.nix

```
@@ -22,12 +22,12 @@ drv.overrideAttrs (oldAttrs: {
     echo "nix-build-cache enabled"
     echo "-*-*-*-*-"
 
-    # Before we do anything, capture the MD5 sums of all source files.
+    # Before we do anything, capture the SHA3 sums of all source files.
     # This is compared against the source files used to produce a cache tarball
     # so we know which files have changed since the cache was made.
     mkdir -p "$out"
     echo "nix-build-cache: Noting current file hashes"
-    find . -type f -print0 | xargs -0 md5sum | sort > $out/MD5SUMS
+    find . -type f -print0 | xargs -0 $ {pkgs.busybox}/bin/sha3sum | sort > $out/SHA3SUMS 
```

Should this maybe be:

```suggestion
    find . -type f -print0 | xargs -0 ${pkgs.busybox}/bin/sha3sum | sort > $out/SHA3SUMS 
```